addons/sourcemod/scripting/tfgo/methodmaps.sp

```diff
@@ -15,6 +15,8 @@ int g_DefaultWeaponIndexes[][] =  {
 int g_PlayerLoadoutWeaponIndexes[TF_MAXPLAYERS + 1][view_as<int>(TFClass_Engineer) + 1][WeaponSlot_BuilderEngie + 1];
 int g_PlayerBalances[TF_MAXPLAYERS + 1];
 Menu g_ActiveBuyMenus[TF_MAXPLAYERS + 1];
+int g_PlayerArmor[TF_MAXPLAYERS + 1][view_as<int>(TFClass_Engineer) + 1];
+bool g_PlayerHelmets[TF_MAXPLAYERS + 1][view_as<int>(TFClass_Engineer) + 1];
```

Comment:
`view_as<int>(TFClassType)` should also work without `+ 1`, though im not sure if that throws legacy enum struct warning